File: 2014/H.java
import java.util.Scanner;

public class H {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int jogadores, partidas, desempenho, gols=0;
		
		jogadores = sc.nextInt();
		partidas = sc.nextInt();
		
		for(int i=0; i<jogadores; i++){
		    int aux=0;
		    for(int j=0; j<partidas; j++){
		        desempenho = sc.nextInt();
		        if(desempenho > 0){
		            aux++;
		        }
		    }
		    if(aux == partidas){
		        gols++;
		    }
		}
		System.out.println(gols);
	}
}

File: 2014/H.c
#include <stdio.h>

int main() {
	int jogadores, partidas, gols=0;
	int desempenho;
	
	while(scanf("%i %i", &jogadores, &partidas)){
		for(int i=0; i<jogadores; i++){
			int aux = 0;
			for(int j=0; j<partidas; j++){
				scanf("%i", &desempenho);
				if(desempenho > 0){
					aux++;
				}
			}
			if(aux == partidas){
				gols++;
			}
		}
		printf("%i", gols);
	}
	return 0;
}
